en/docs/administer/monitoring-server-health.md

```diff
@@ -154,3 +193,47 @@ To deploy it, copy the `         .jar        ` file to the
 the OSGI bundle and paste it in the
 `         <IS_HOME>/repository/component/dropins/        `
 directory.
+
+Then register the new health checker as below in the 'deployment.toml'
+file if needed.
+
+```toml
+[[health_checker]]
+name = "customChecker" 
+order="87" 
+[health_checker.properties] 
+property1 = "property-1-value"
+property2 = "property-2-value" 
+```
+
+
+??? Tip 
+    A sample configuration section of `deployment.toml` related to
+    health checkers can be as below. 
```

Comment:
Please change to "can be as below" to "...**is shown below**"

Reply:
fixed